CMakeLists.txt

```diff
@@ -53,8 +53,6 @@ configure_file("${CMAKE_CURRENT_SOURCE_DIR}/core/version.h.in" "${CMAKE_CURRENT_
 
 if(ANDROID)
     add_library(${PROJECT_NAME} SHARED core/nullDC.cpp)
-    set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_RELEASE} -O3")
-    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_RELEASE} -O3")
```

Comment:
I guess this was done because of the lack of release builds, that's why I removed it here.